main.cpp

```diff
@@ -0,0 +1,129 @@
+#include <iostream>
+#include <string>
+#include <stack>
+#include <vector>
+
+/**
+ * @brief Get and pop last two items on the stack
+ *
+ * @param stack
+ * @param a
+ * @param b
+ */
+void getTwoTopItems(std::stack<int> &stack, int &a, int &b)
+{
+    a = stack.top();
+    stack.pop();
+    b = stack.top();
+    stack.pop();
+}
+
+int main()
+{
+    std::stack<int> stack = std::stack<int>();
+    bool working = true;
+    std::string input;
+
+    std::vector<std::string> options = std::vector<std::string>{"add", "sub", "mul", "div", "exit", "peak", "swap", "pop"};
+
+    std::cout << "add, sub, mul, div, exit, peak, swap, pop, or the number to add: " << std::endl;
+    do
```

Comment:
doesnt matter, but i prefer while loops instead of do while, personal preference